src/yz_index_hashtree.erl

```diff
@@ -526,9 +530,15 @@ close_trees(S=#state{trees=Trees, closed=false}) ->
     Trees2 = [begin
                   NewTree =
                       try
-                          hashtree:flush_buffer(Tree)
-                      catch _:_ ->
-                              lager:warning("Failed to flush trees during close"),
+                          case hashtree:next_rebuild(Tree) of
+                              full -> hashtree:flush_buffer(Tree);
```

Comment:
Add comment to say deliberately not marking cleanly closed to avoid the cost of a full rebuild on shutdown.  Next restart will put it back in full state.